src/handlers/predict.py

```diff
@@ -1,32 +1,19 @@
 from ..entities.prediction_request import PredictionRequestPydantic
 from ..helpers import model_decoder, json_to_predreq
 
-
 def model_post_handler(request: PredictionRequestPydantic):
     model = model_decoder.decode(request.model['rawModel'])
-    data_entry_all = json_to_predreq.decode(request)
-    _ = model(data_entry_all)
-
-    if isinstance(model.prediction[0], list):
-        results = {model.Y[i]: [item[i] for item in model.prediction] for i in range(len(model.prediction[0]))}
-    elif isinstance(model.prediction, list):
-        if isinstance(model.Y, list):
-            results = {model.Y[0]: [item for item in model.prediction]}
-        else:
-            results = {model.Y: [item for item in model.prediction]}
+    data_entry_all = json_to_predreq.decode(request, model)
+    prediction = model.predict_onnx(data_entry_all)
+    if model.task == 'classification':
+        probabilities = model.predict_proba_onnx(data_entry_all)
     else:
-        results = {model.Y: [item for item in model.prediction]}
+        probabilities = [None for _ in range(len(prediction))]
 
-    if model.doa:
-        results['AD'] = model.doa.IN
-    else:
-        results['AD'] = [None for _ in range(len(model.prediction))]
-
-    if model.probability:
-        results['Probabilities'] = [list(prob) for prob in model.probability]
-    else:
-        results['Probabilities'] = [[] for _ in range(len(model.prediction))]
+    results = {model.dependentFeatures[0]['name']: [str(item) for item in prediction]}
```

Comment:
This should be
```suggestion
    results = {model.dependentFeatures[0]['key']: [str(item) for item in prediction]}
```

Also what happens if the model has more than 1 labels/predictedFeatures?